docs/drafts/derive-serviceaccount.md

```diff
@@ -0,0 +1,401 @@
+# Derive minimal ServiceAccount required for ClusterExtension Installation and Management
+
+OLMv1 does not provide cluster admin privileges by default for installing cluster extensions. This means that the installation process will require a service account with sufficient privileges to install the bundle. It depends on the cluster extension developer to specify the exact permissions required for the management of any specific bundle. A ServiceAccount needs to be explicitly specified for installing and upgrading operators to avoid errors when deploying your cluster extension.
+
+The ServiceAccount is specified in the ClusterExtension manifest as shown below:
+
+```yaml
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterExtension
+metadata:
+  name: argocd
+spec:
+  source:
+    sourceType: Catalog
+    catalog:
+      packageName: argocd-operator
+      version: 0.6.0
+  install:
+    namespace: argocd
+    serviceAccount:
+      name: argocd-installer
+```
+
+The initial stable version (v1.0.0) only supports FBC catalogs containing registry+v1 bundles. OLMv1 will not support all OLMv0 content. OLMv1 will only support bundles that meet the following criteria:
+* AllNamespaces install mode is enabled
+* No dependencies on other packages or GVKs
+* No webhooks
+* Does not make use of the OperatorConditions API
+
+### Required RBAC
+
+The cluster extension installer should have the prerequisite permissions in order to be able to assign the controller the RBAC it requires. In order to derive the minimal RBAC for the installer service account, you must specify the following permissions:
```

Comment:
This first sentence is a bit hard to parse and I think can just be removed as well, since it's basically just saying you need to properly configure the SA which is what this whole doc is about.

Reply:
Also the second sentence says "In order to derive.. you must specify" which isn't exactly true. Deriving the permissions means figuring out what they should be, but you don't figure out what they should be by specifying what they should be. Something more like "The service account must have the following permissions, which you can determine by referencing the bundle of the ClusterExtension you want to install:"

Reply:
addressed